docs/usermanual/userman_admin.rst

```diff
@@ -645,14 +645,23 @@ You can also run the ``kalite`` with a completely different Python settings modu
 Changing base settings
 ^^^^^^^^^^^^^^^^^^^^^^
 
-By default, ``~/.kalite/settings.py`` will load ``kalite.project.settings.base`` which are the basic settings. But you can also load Raspberry Pi settings by changing the file to read something like:
+By default, ``~/.kalite/settings.py`` will load ``kalite.project.settings.base`` which are the basic settings. But you can also load Raspberry Pi settings by changing the file to read something like::
 
-  .. ::
-    from kalite.project.settings.raspberry_pi import *
-    # Put your settings here, e.g.
-    # MY_SETTING_VAR = 123
+  from kalite.project.settings.raspberry_pi import *
+  # Put your settings here, e.g.
+  # MY_SETTING_VAR = 123
```

Comment:
But MY_SETTING_VAR is not a valid setting :) I mean, if the user doesn't know what `#` means in Python, then they are lucky to even be able to open the file in a text editor :)